src/structures/GuildMember.js

```diff
@@ -136,6 +145,27 @@ class GuildMember extends Base {
     return this.guild.voiceStates.cache.get(this.id) || new VoiceState(this.guild, { user_id: this.id });
   }
 
+  /**
+   * A link to the member's server avatar.
+   * @param {ImageURLOptions} [options={}] Options for the Image URL
+   * @returns {?string}
+   */
+  avatarURL({ format, size, dynamic } = {}) {
+    if (!this.avatar) return null;
+    return this.client.rest.cdn.Avatar(this.id, this.avatar, format, size, dynamic);
+  }
+
+  /**
+   * A link to the member's server avatar if they have one.
+   * Otherwise a link to their user display avatar will be returned.
+
+   * @param {ImageURLOptions} [options={}] Options for the Image URL
```

Comment:
```suggestion
   * Otherwise a link to their user display avatar will be returned.
   * @param {ImageURLOptions} [options={}] Options for the Image URL
```
Remove the empty line here